scripts/install-protoc.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# Copyright 2024 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This script ensures the installation of protobuf on client machine.
+# In case of manual run of this script, make sure you pass the args
+# expected at
+# https://github.com/grpc/grpc-go/blob/master/scripts/install-protoc.sh#L60
```

Comment:
This feels very prone to move around.  In fact, line 60 of this file is something to detect your architecture.  Maybe we can do something better?

Reply:
I've updated the top-level comment to talk more about usages. PTAL